peachjam/models/ingestors.py

```diff
@@ -26,13 +27,29 @@ def __str__(self):
 
 
 class Ingestor(models.Model):
+
+    ONE_MINUTE = (60, "one minute")
+    FIVE_MINUTES = (5 * 60, "five minutes")
+    THIRTY_MINUTES = (30 * 60, "thirty minutes")
+    INGESTOR_REPEAT_CHOICES = (
+        ONE_MINUTE,
+        FIVE_MINUTES,
+        THIRTY_MINUTES,
+        *Task.REPEAT_CHOICES,
+    )
+
     adapter = models.CharField(_("adapter"), max_length=2048)
     last_refreshed_at = models.DateTimeField(
         _("last refreshed at"), null=True, blank=True
     )
     name = models.CharField(_("name"), max_length=255)
     enabled = models.BooleanField(_("enabled"), default=True)
 
+    repeat = models.BigIntegerField(choices=INGESTOR_REPEAT_CHOICES, default=Task.NEVER)
+    schedule = models.BigIntegerField(

```

Comment:
Now that we know that these two must be different, it's good to have them separate. So we can configure `schedule` to be a small value like 10 minutes, and then repeat can be daily. That will ensure it's run once and then repeats. It may run more often than we need, but that's better than not running.

Reply:
yeah, I've updated this with the new queue_task method